service/src/main/kotlin/fi/espoo/evaka/application/ApplicationControllerV2.kt

```diff
@@ -786,6 +793,10 @@ data class PlacementProposalConfirmationUpdate(
     val otherReason: String?,
 )
 
+data class AcceptPlacementProposalRequest(
+    val rejectReasons: Map<PlacementPlanRejectReason, String>
```

Comment:
Tämän voisi nimetä uudelleen:
```suggestion
    val rejectReasonTranslations: Map<PlacementPlanRejectReason, String>
```

Reply:
:+1: